docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+
+= Resolving issues with the {ProductName} extension
+
+You can resolve issues by using a Quick Fix or by editing the code yourself.
+
+.Procedure
+
+. Resolve issues detected by the {ProductName} extension by performing one of the following actions:
+
+* If the issue displays the Quick Fix icon (image:vs_optional.png[Optional or Quick Fix]), you can right-click the issue and select *Apply All QuickFixes*.
+
+* If the issue has any other icon or you decide not to use all suggested Quick Fixes, right-click the issue, select *Open Code*, make any changes needed to the code, and then save the file.
```

Comment:
You can remove the first clause. It's irrelevant. Just start with "Right-click ..."